File: 70 Climbing Stairs/Climbing Stairs.js
/**
 * @param {number} n
 * @return {number}
 */
const climbStairs = (n) => {
  let low = 0;
  let high = 1;

  for (let i = 1; i <= n; i++) {
    const temp = high;
    high = high + low;
    low = temp;
  }

  return high;
};

File: 70 Climbing Stairs/Climbing Stairs.java
class Solution {
    public int climbStairs(int n) {
        int low = 0;
        int high = 1;

        for (int i = 1; i <= n; i++) {
            int temp = high;
            high = high + low;
            low = temp;
        }

        return high;
    }
}
